docs/rendering_system.rst

```diff
@@ -17,8 +17,8 @@ to `PEP 676 <https://peps.python.org/pep-0676/>`__.
 
 Configuration is stored in three files:
 
-- ``conf.py`` contains the majority of the Sphinx configuration
-- ``contents.rst`` contains the compulsory table of contents directive
+- ``peps/conf.py`` contains the majority of the Sphinx configuration
```

Comment:
Thoughts on keeping the conf.py file in the repository root? That'll keep things unmixed more completely.

Reply:
This was my first attempt, but it requires patching even more of the HTML builder than at present. Sphinx would want to write output files to ``build/dirhtml/peps/pep-XXXX.html``, so we'd need to hoist that, but then also do post processing to rewrite all relative links, make script/link tags work correctly, etc.

It is do-able, but I think better to tackle later on.

Reply:
> Here's a working patch

Thanks for this, helps to explain my misunderstanding. I thought you were advocating also moving ``contents.rst`` up to the root. I think that position (having only PEPs in the PEPs folder) would be nice to achieve, but I'd like to do the entire thing---only having ``conf.py`` in the root feels a bit odd.

A